lib/ansible/modules/storage/netapp/na_ontap_service_processor_network.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/python
+
+# (c) 2018, NetApp, Inc
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: na_ontap_service_processor_network
+short_description: Manage NetApp Ontap service processor network
+extends_documentation_fragment:
+    - netapp.na_ontap
+version_added: '2.6'
+author:
+- Chris Archibald (carchi@netapp.com), Kevin Hutton (khutton@netapp.com)
+description:
+- Modify a Ontap service processor network
+options:
+  state:
```

Comment:
any reason for this arg to exist?

Reply:
Only for consistency.   Not sure it's a good enough reason.